grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -122,6 +124,7 @@ class Requisition implements Comparable<Requisition>, Serializable {
         id generator: 'uuid'
         requisitionItems cascade: "all-delete-orphan", sort: "orderIndex", order: 'asc', batchSize: 100
 
+        statusSortOrder formula: RequisitionStatus.getStatusSortOrderFormula("status")
```

Comment:
Why do we need to pass status? Is there ever going to be another value? If not I'm ok moving this in to the method to make it more readable.

Reply:
@jmiranda I wanted it to be flexible if the RequisitionStatus would be used in other domains with different naming. I guess I can remove it for now.

Reply:
@jmiranda done